code/game/objects/effects/contraband.dm

```diff
@@ -414,6 +414,16 @@
 	desc = "A poster advertising the newest drink \"Buzz Fuzz\" with its iconic slogan of ~A Hive of Flavour~."
 	icon_state = "poster45"
 
+/obj/structure/sign/poster/contraband/scum
+	name = "Security are Scum"
+	desc = "Anti-security propaganda. Features a human NanoTrasen security officer being shot in the head, with the words 'Scum' and a short inciteful manifesto. Used to anger security."
+	icon_state = "poster46"
+
+/obj/structure/sign/poster/contraband/scum
+	name = "Security are Scum"
+	desc = "Anti-security propaganda. Features a human NanoTrasen security officer being shot in the head, with the words 'Scum' and a short inciteful manifesto. Used to anger security."
+	icon_state = "poster47"
```

Comment:
Same description, same name, same path, just different icon states.
I believe the second is pertinent to something else and not security though.